docs/language/accounts.mdx

```diff
@@ -88,6 +91,9 @@ to the `prepare` phase of the transaction.
 
       let keys: AuthAccount.Keys
 
+      // All the paths associated with this account
+      let paths: [Path]
```

Comment:
Would it be better to group these 'paths' related operations (this one and the [iterator functions](https://github.com/onflow/cadence/pull/1851)) together? I am thinking of something similar to `keys` field (of type `AuthAccount.Keys`).

e.g:
```cadence
let paths: AuthAccount.Paths
```
and `AuthAccount.Paths` to be something like:
```cadence
struct Paths {
    pub allPaths: [Path]

    pub fun forEachPublic(_ function: ((PublicPath, Type): Bool))

    pub fun forEachPrivate(_ function: ((PrivatePath, Type): Bool))

    pub fun forEachStored(_ function: ((StoragePath, Type): Bool))
}
```
The advantage is:
 - It groups simimlar operations toegther
 - The usage is more verbose (specially for the iterator functions):
    ```cadence
    account.Paths.forEachPublic(...)
    ```

Reply:
This is just my POV. I'm open to other suggestions, and certainly can ask what others think.

Reply:
I am ambivalent about it either way; curious what the community thinks.

Reply:
I cross-posted [here](https://github.com/onflow/cadence/issues/208#issuecomment-1199840213). Maybe the issue itself is a better place to discuss the design, etc.